src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -430,6 +433,8 @@
     text: az aks create -g MyResourceGroup -n MyManagedCluster --node-osdisk-diskencryptionset-id <disk-encryption-set-resource-id>
   - name: Create a kubernetes cluster with ephemeral OS enabled.
     text: az aks create -g MyResourceGroup -n MyManagedCluster --node-osdisk-type Ephemeral --node-osdisk-size 48
+  - name: Create a kubernetes cluster with custom tags
+    text: az aks create -g MyResourceGroup -n MyManagedCluster --tags "a=b" "c=d"
```

Comment:
maybe foo=bar or examples of actual tags would be good

Reply:
fixed, thanks